src/pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -613,7 +613,11 @@ def _check_strategy_and_fallback(self) -> None:
                 f"You selected strategy to be `{DDPFullyShardedNativeStrategy.strategy_name}`, "
                 "but GPU accelerator is not used."
             )
-
+        if strategy_flag in ("ddp_fork", "ddp_fork_find_unused_parameters_false") and not hasattr(os, "fork"):
```

Comment:
Do we need to hardcode these options again? Can't we extract them from the registry?

Reply:
Hmm, it would be nice if we could avoid it, but there is no elegant way to do this right now. We could add a method to the StrategyRegistry that returns us a view of the registered items for a particular class only. 

Or we could naively filter the keys for the substring "fork", if that's what you meant?

Reply:
> We could add a method to the StrategyRegistry that returns us a view of the registered items for a particular class only.

This is what I meant. But fine to leave it for latter if you prefer.

Reply:
@kaushikb11 A different error would show up when attempting to launch with .fit(), pointing that the start method is not supported. We can generalize and improve the error handling further, but I think this is good enough and catches the most common use cases for 99% of the users. I will leave this as fine tuning for a follow up.